main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/google/go-github/github"
+	"github.com/joho/godotenv"
+	"golang.org/x/oauth2"
+)
+
+const (
+	GithubToken = "GITHUB_TOKEN"
+)
+
+func main() {
```

Comment:
Can me make some code, so this line doesn't have to be exists?
Why, we can make sure that it will be there, by coding it will
be there..